app/assets/javascripts/dictionary.js.coffee

```diff
@@ -18,3 +18,28 @@ doTestField = ->
 
 $ ->
   $('#field_name_test').on('input', doTestField)
+
+
+# Sets up the sidenav on the dictionary.
+# Must be called on the page, or else the script misbehaves
+window.dictionarySetupSidenav = ->
```

Comment:
I did try that at first. But the navigation menu would appear misplaced if you use a anchored URL (like http://localhost:3000/dictionary#addresses).

I think this happens because of the timing of running the script and rendering the menu.
Since the script needs the position of the menu, it needs to be run after the menu is rendered.

I will try to make it some other way. Will post back soon.